### DISCUSSION TITLE
Question(mux-player-react): Customize fullscreen behavior?

### DISCUSSION THREAD

**Discussion by vojtaholik** [muxinc/elements#532] [Q&A]
we’d like to display a video overlay that can be interacted with when in fullscreen mode.

is there a way to customize default fullscreen toggle behavior? we’d like to display a video overlay that can be interacted with when in fullscreen mode. to achieve that, I thought it would make sense for the fullscreen toggle to target an outer element containing overlay content instead of just the player. I put together an example sandbox showcasing this that uses a custom fullscreen toggle button: [https://codesandbox.io/p/sandbox/silly-einstein-ruoge4?file=%2Fpages%2Findex.tsx](https://codesandbox.io/p/sandbox/silly-einstein-ruoge4?file=%2Fpages%2Findex.tsx)

is there a way to customize the fullscreen toggle button that is part of the player’s UI? I tried listening for `fullscreenchange` event on MuxPlayer ref and calling `preventDefault()` but that doesn’t seem to be working and it keeps falling back to its default behavior.

or would you suggest a different solution? youtube has a similar feature with their "next up overlay" after finishing a video.

any help is welcome! 🙏 

https://user-images.githubusercontent.com/25487857/210329744-9905ab65-8764-400d-8061-ee886bca59a3.mp4

**Comment by cjpillsbury** [comment]
Hey @vojtaholik! Currently working to try to smuggle in a feature in Media Chrome (which Mux Player is built on top of) to support this! Will keep ya posted.

relates to https://github.com/muxinc/media-chrome/pull/415

**Reply by vojtaholik** [reply]
Love that, thanks!
